torch/onnx/utils.py

```diff
@@ -72,6 +72,7 @@ def is_in_onnx_export() -> bool:
 # TODO(justinchuby): Remove dependency to this global variable from constant_fold.cpp
 # Skip check due to cannot import IValue from torch._C
 _params_dict = {}  # type: ignore[var-annotated]
+_custom_onnxscript = list()  # type: ignore[var-annotated]
```

Comment:
I hope we can find a cleaner way to store the state than using a global var. Do we have options?

Reply:
Yes, I will reference on registry and see if we can have one data structure for onnx-script function.